libwallet/src/slatepack/types.rs

```diff
@@ -195,14 +195,16 @@ impl Slatepack {
 		b.copy_from_slice(&result[0..32]);
 
 		let x_dec_secret = StaticSecret::from(b);
-		let key = age::keys::SecretKey::X25519(x_dec_secret);
+		let x_dec_secret_bech32 =
+			bech32::encode("age-secret-key-", (&x_dec_secret).to_bytes().to_base32())?;
```

Comment:
Just want to double check that this bech32 string can be cast as an age `Identity` in the next line without calling `.to_uppercase()` on the bech32 string, as iirc age library expects this string to be uppercase.

Reply:
After taking a closer look, this was changed to a lower case HRP so the code here is correct
https://github.com/str4d/rage/blob/456ce707f6db01f2f953c9e22c058964f3437795/age/src/x25519.rs#L20